packages/node_modules/@ciscospark/internal-plugin-board/src/board.js

```diff
@@ -245,7 +265,7 @@ const Board = SparkPlugin.extend({
           type: contentType,
           encryptionKeyUrl
         },
-          pick(res, `file`, `payload`)
+          pick(res, [`file`, `payload`])
```

Comment:
Generally avoid the array option when passing literals. it's an extra array allocation that doesn't add value.